lib/rules/no-new-nonconstructor.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @fileoverview Rule to disallow use of the new operator with global no constructor functions
+ * @author Sosuke Suzuki
```

Comment:
Just a question, most of the code for this rule is the same as `no-new-symbol`. Who should we mention as a an author?

Reply:
Yes, that would be nice.